web/themes/contrib/civictheme/includes/primary_navigation.inc

```diff
@@ -31,5 +29,8 @@ function civictheme_preprocess_block__menu_block__civictheme_primary_navigation(
 
   if (str_starts_with($region, 'sidebar')) {
     $variables['in_sidebar'] = TRUE;
+    $expand_all_items = $variables['configuration']['expand_all_items'] ?? FALSE;
```

Comment:
Added override only for sidebar regions, (Header navigation wont be effected - they will still follow individual menu item settings)